iOS/swiftui-oidc/PingExample/PingExample/UserProfileViewModel.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  UserProfileViewModel.swift
+//  PingExample
+//
+//  Created by jey periyasamy on 5/8/24.
+//
+
+import Foundation
+
+class UserProfileViewModel: ObservableObject {
+    
+    @Published var userProfile: String = ""
+    
+    init() {
+        Task {
+            await self.accessToken()
+        }
+    }
+    
+    func accessToken() async {
+//        let foo = await davinci.user()?.userinfo(cache: <#T##Bool#>)
```

Comment:
Can we provide details here why this is commented out?

Reply:
No idea...left over I guess...I will remove